examples/cdi-request-scope/README_pt.adoc

```diff
@@ -0,0 +1,161 @@
+= CDI @RequestScoped
+:index-group: CDI
+:jbake-type: page
+:jbake-status: published
+
+Este exemplo mostra o uso da anotação `@RequestScoped` para objetos injetados. Um objeto que é 
+definido como `@RequestScoped` é criado uma vez para cada requisição e é compartilhado por todos os 
+beans que o injetam através do mesmo request.
+
+== Exemplo
+
+Este exemplo retrata um cenário similar ao cdi-application-scope. O visitante de um restaurante 
+pede uma sopa ao garçom. O pedido é passado para o chefe que prepara o prato e o devolve ao garçom 
+que entrega para o visitante.
+
+=== Waiter (garçom)
+
+O `Waiter` session bean recebe uma requisição através do método `orderSoup()` da classe de teste.
+Uma instancia de `Soup` sera criada nesse método e ela sera compartilhada em toda a requisição com o 
```

Comment:
Hi Gabriel, in the sentence: "Uma instancia de `Soup` sera criada" the word "instancia" have an accent "instância", and the word "sera" is "será".

Reply:
done